### PR TITLE
Feature/ update archetype feature locations

### PR DESCRIPTION
**Pull request by annacv**
[EX-7078](https://searchbroker.atlassian.net/browse/EX-7078)
# Pull request template
 

- Update x-components to `3.0.0-alpha.183 version`
- Implement `queryFeature` prop in `CustomQueryPreview` component

## Motivation and context

Update archetype with feature/locations in NQs and Queries preview

- [x] Dependencies. If any, specify: x-components to `3.0.0-alpha.183 version`
- [ ] Open issue. If applicable, link:

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [ ] Change requires a documentation update

## What is the destination branch of this PR?

- [x] `Main`
- [ ] Other. Specify:

## How has this been tested?

- Check in VueDevTools that `QueryPreview` component has the `QueryFeature` prop defined
- When X is opened, the request of the `no-search` carrousels is sent with the origin `customer:no-query`.
- Clicking in `no-search` carrousels' `view-more` button fires a new search request with the origin `next_query:results`.
- Clicking in `next-queries`'s `view-more` button in the results grid fires a new search request with the origin `next_query:results`

Tests performed according to [testing guidelines](https://github.com/empathyco/x/blob/main/.github/contributing/tests.md): 

## Checklist:

- [x] My code follows the **[style guidelines](https://github.com/empathyco/x/blob/main/.github/CONTRIBUTING.md#style-guides)** of this project.
- [x] I have performed a **self-review** on my own code.
- [ ] I have **commented** my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate **no new warnings**.
- [ ] I have added **tests** that prove my fix is effective or that my feature works.
- [ ] New and existing **unit tests pass locally** with my changes.

### CODE REVIEW

**Review by LuisMartinez15 — APPROVED** [review]
good job!

File: src/components/pre-search/custom-query-preview.vue [hunk 1]
```diff
@@ -87,6 +89,9 @@
     }
   })
   export default class CustomQueryPreview extends Vue {
```
Comment (javieri-empathy) [inline src/components/pre-search/custom-query-preview.vue:91]: Right now this is sending the `origin=customer:results` parameter to the search.

image


 The location should be `no_query`. To do so, in `main.vue` you can wrap the `CustomQueryPreview` component with a `LocationProvider` component.

**Review by javieri-empathy — CHANGES_REQUESTED** [review]

File: src/components/main.vue [hunk 2]
```diff
@@ -1,6 +1,10 @@
 <template>
   <div class="x-list x-list--vertical">
-    <CustomQueryPreview :class="$x.device === 'mobile' ? 'x-margin--top-05' : 'x-margin--top-10'" />
+    <LocationProvider location="no-query">
```
Comment (javieri-empathy) [inline src/components/main.vue:3]: It's `no_query` 😅
```suggestion
    <LocationProvider location="no_query">
```

**Review by LuisMartinez15 — CHANGES_REQUESTED** [review]
check javi's comment

**Review by javieri-empathy — APPROVED** [review]

**Review by LuisMartinez15 — APPROVED** [review]

**Pull request merged**